setup.py

```diff
@@ -37,6 +38,7 @@
             'config = config.main:cli',
             'sfputil = sfputil.main:cli',
```

Comment:
I believe you also need to add entry points here under `console_scripts` for `debug` and `undebug`

Reply:
Right, will add those two right away.